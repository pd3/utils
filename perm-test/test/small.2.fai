1	5000
2	5000
3	5000
