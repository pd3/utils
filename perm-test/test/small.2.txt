

./perm-test -t test/small.2.tgt.txt -b test/small.2.bg.txt -f test/small.2.fai -c test/small.2.calls.txt -n 1e9
./perm-test -t test/small.2.tgt.txt -b test/small.2.bg.txt -f test/small.2.fai -c test/small.2.calls.txt -n 1e9 --no-bg-overlap

    fai:
        1	5000
        2	5000
        3	5000
    bg:
        1	1000	1100
        1	1500	1600
        1	2000	2100
        2	1000	1100
        2	1500	1600
        2	2000	2100
        3	1000	1100
        3	1500	1600
        3	2000	2100
    tgt:
        1	1300	1400
        1	1900	2000
        2	1300	1400
        2	1900	2000
        3	1300	1400
        3	1900	2000

    calls:
        1350	1350
        1351	1351

    exact: (3*(1400-1300+1 + 2000-1900+1)*1./3/(1100-1000+1 + 1600-1500+1 + 2100-2001+1 + 1400-1300+1 + 2000-1900+1))**2 = 0.4007936507936508**2 = 0.1606355505165029
    perm-test:  0.160611462
    err: 2.40885e-05


