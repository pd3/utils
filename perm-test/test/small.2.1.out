BG	1	1000	1100
TGT	1	1300	1400
BG	1	1500	1600
TGT	1	1900	2000
BG	1	2001	2100
BG	2	1000	1100
TGT	2	1300	1400
BG	2	1500	1600
TGT	2	1900	2000
BG	2	2001	2100
BG	3	1000	1100
TGT	3	1300	1400
BG	3	1500	1600
TGT	3	1900	2000
BG	3	2001	2100
MSG	Maximum chromosome randomization error due to length discretization: 1.5e-03%
CALL	TGT	1	1350	1350
CALL	TGT	1	1351	1351
NCALLS	2	0
